docs/source/essentials/schema.md

```diff
@@ -17,55 +30,208 @@ type Book {
+## Introspection and documentation
 
-## Introspection
+Introspection is an optional feature, enabled by default during development, which allows clients to explore the types a GraphQL.
 
-Introspection is an automatic benefit built into the GraphQL specification which allows users to ask a server what operations it supports.  This facilitates SDL-generation since GraphiQL and other tools will can provide you specific insight into the fields available at each level of a GraphQL operation.  Protecting data exposed by a GraphQL schema is important and more information on security can be found in [security best practices]().
+Furthermore, by using (optional) string literals within the schema, it's possible to explain exactly what a field does:
 
+```graphql
+type Book {
+  "Returns the title of the book, as listed in the card catalog."
+  title: String
+}
+
+type Query {
+  "Fetch a list of our extensive book collection!"
+  getBooks: [Book]
+}
+```
+
+This makes SDL-generation even easier since many GraphQL tools auto-complete the field names (along with descriptions, when available) at each level of a GraphQL operation.
+
+## Operation names
+
+When sending the queries and mutations in the above examples, we've used either `query { ... }` or `mutation { ... }` respectively.  While this is fine, and particularly convenient when running queries by hand, it makes sense to name the operation in order to quickly identify operations during debugging or to aggregate similar operations together for application performance metrics, for example, when using [Apollo Engine]() to monitor an API.
+
+Operations can be named by placing a name after the `query` or `operation` keyword, as in:
+
+```graphql
+query HomepageBookListing {
+  getBooks {
+    title
+  }
+}
+
+mutation HomepageQuickAddBook {
+  addBook(title: "Fox in Socks", author: "Dr. Seuss") {
+    title
+  }
+}
+```
 
 ## SCRATCHPAD
 
-* This section ties all of the information in the prereqs to show you how to implement Queries with the Apollo Server
-  * essentially copy and paste code that you can then add onto
+### Query
+  * GraphQL query defines the shape of data that will be returned by a particular request
+    * make sure it has arguments
+  * This query is then checked again the server's schema
+    * looks like this:
 
-* Mutations exist because they have special argument types called Input types
-* Input types only contain scalar types and cannot have any other input types
-  * ensures that data from the client is always serializable and we don't lose any information, since circular references don't survive network call
+### Mutation
+  * Mutations exist because they have special argument types called Input types
+  * Input types only contain scalar types and cannot have any other input types
+    * ensures that data from the client is always serializable and we don't lose any information, since circular references don't survive network call
```

Comment:
I didn't go into this here, but I'd welcome suggestions on how to incorporate this here (or alternatively, if we should just be elaborating on this elsewhere).

Reply:
I was wrong. Input types don't actually do this, since they can contain other input types 🤦‍♂️